Orange-OpenSource/its-client
# Draft: CPM v1.1.3

New features
-----------------

* **schema**: Provide CPM schema version 1.1.3
* **rust**: apply changes to the data concerned structures: `CollectivePerceptionMessage`, `PerceivedObject`

Closes #34 

How to test
---------------

1. Check the schema validity
    **=> it must be valid JSON**
    **=> it must be valid against its own schema `"https://json-schema.org/draft/2020-12/schema"`**
2. Build and run unit tests
    ```
    cargo clean; cargo build --all; cargo test --all
    ```
    **=> Build must be successfull**
    **=> All tests must pass**
3. Run the example application
	 ```
	 cargo run --bin its-client -- --mqtt-host france1 --mqtt-port 11883 --mqtt-username <username> --mqtt-password <password> --mqtt-client-id ora_itsclient_test
	 ```
	 **=> Binary must run and behave correctly**
